build/packages-template/bbb-html5/build.sh

```diff
@@ -12,117 +12,51 @@ BUILD=$1
-# /usr/share/meteor/bundle/programs/server/node_modules
-npm i
-cd -
-cp -r /tmp/html5-build/bundle staging/usr/share/meteor
+npm ci --production
+DISABLE_ESLINT_PLUGIN=true npm run build
 
-# generate index.json locales file if it does not exist
-if [ ! -f staging/usr/share/meteor/bundle/programs/web.browser/app/locales/index.json ]; then
-  find staging/usr/share/meteor/bundle/programs/web.browser/app/locales -maxdepth 1 -type f -name "*.json" -exec basename {} \; | awk 'BEGIN{printf "["}{printf "\"%s\", ", $0}END{print "]"}' | sed 's/, ]/]/' > staging/usr/share/meteor/bundle/programs/web.browser/app/locales/index.json
-fi
-
-cp workers-start.sh staging/usr/share/meteor/bundle
-chmod +rx staging/usr/share/meteor/bundle/workers-start.sh
-
-cp mongod_start_pre.sh staging/usr/share/meteor/bundle
-chmod +rx staging/usr/share/meteor/bundle/mongod_start_pre.sh
-
-cp mongo-ramdisk.conf staging/usr/share/meteor/bundle
-# cp mongo-ramdisk.conf staging/etc/mongod.conf
-
-mkdir -p staging/usr/lib/systemd/system
-cp bbb-html5.service staging/usr/lib/systemd/system
-cp disable-transparent-huge-pages.service staging/usr/lib/systemd/system
-
-mkdir -p staging/usr/share
-
-# replace v=VERSION with build number in head and css files
-if [ -f staging/usr/share/meteor/bundle/programs/web.browser/head.html ]; then
-  sed -i "s/VERSION/$(($BUILD))/g" staging/usr/share/meteor/bundle/programs/web.browser/head.html
-fi
+cp -r dist/* staging/var/bigbluebutton/html5-client
 
-find staging/usr/share/meteor/bundle/programs/web.browser -name '*.css' -exec sed -i "s/VERSION/$(($BUILD))/g" '{}' \;
-
-# Compress CSS, Javascript and tensorflow WASM binaries used for virtual backgrounds. Keep the
-# uncompressed versions as well so it works with mismatched nginx location blocks
-find staging/usr/share/meteor/bundle/programs/web.browser -name '*.js' -exec gzip -k -f -9 '{}' \;
-find staging/usr/share/meteor/bundle/programs/web.browser -name '*.css' -exec gzip -k -f -9 '{}' \;
-find staging/usr/share/meteor/bundle/programs/web.browser -name '*.wasm' -exec gzip -k -f -9 '{}' \;
-
-mkdir -p staging/etc/nginx/sites-available
-cp bigbluebutton.nginx staging/etc/nginx/sites-available/bigbluebutton
-
-mkdir -p staging/usr/share/bigbluebutton/nginx
-cp sip.nginx staging/usr/share/bigbluebutton/nginx
-
-mkdir -p staging/var/www/bigbluebutton
-touch staging/var/www/bigbluebutton/index.html
+##
 
 . ./opts-$DISTRO.sh
 
-#
-# Build RPM package
 fpm -s dir -C ./staging -n $PACKAGE \
     --version $VERSION --epoch $EPOCH \
     --before-install before-install.sh \
-    --after-install after-install.sh \
-    --before-remove before-remove.sh \
-    --after-remove after-remove.sh \
-    --description "The HTML5 components for BigBlueButton" \
+    --after-install after-install.sh        \
+    --after-remove  after-remove.sh        \
+    --depends unzip                        \
+    --description "BigBlueButton bbb-html5-client" \
```

Comment:
Do we actually introduce `bbb-html5-client`? Please consider something more descriptive.